directory/directory_transport.go

```diff
@@ -190,8 +190,3 @@ func (ref dirReference) signaturePath(index int, instanceDigest *digest.Digest)
 	}
 	return filepath.Join(ref.path, fmt.Sprintf("signature-%d", index+1)), nil
 }
-
-// versionPath returns a path for the version file within a directory using our conventions.
-func (ref dirReference) versionPath() string {
```

Comment:
Keeping the `"version"` path logic in this place, beside the other `…Path` locations, might be a good way to sort-of document the file format? Also, eventually, https://github.com/containers/image/issues/1876 will need this to live outside of `initDestDir`.